### PR TITLE
fix(proteus): Specifying "sodium.base64_variants.ORIGINAL"

### PR DESCRIPTION
**Pull request by bennyn**

### CODE REVIEW

File: packages/cryptobox/test/CryptoboxSpec.common.js [hunk 1]
```diff
@@ -175,21 +177,37 @@ describe('cryptobox.Cryptobox', () => {
...
+      });
+    });
+  });
+
   describe('Sessions', () => {
     let box = undefined;
     const sessionIdUnique = 'unique_identifier';
 
-    beforeEach(done => {
+    beforeEach(async done => {
```
Comment (ffflorian) [inline packages/cryptobox/test/CryptoboxSpec.common.js]: No need for `async` here

Comment (bennyn) [reply packages/cryptobox/test/CryptoboxSpec.common.js]: But `Proteus.keys.IdentityKeyPair.new` is async.

Comment (ffflorian) [reply packages/cryptobox/test/CryptoboxSpec.common.js]: But you have `async` here: https://github.com/wireapp/wire-web-packages/pull/272/files#diff-f3493822b0bf4f72fb4ebffff1c3e2acR203

**Comment by ffflorian** [comment]
You forgot a parameter in [PreKeyBundleSpec.js#L116](https://github.com/wireapp/wire-web-packages/blob/cryptobox-tests/packages/proteus/spec/keys/PreKeyBundleSpec.js#L116). :slightly_smiling_face:

**Pull request merged**